### DISCUSSION TITLE
commit help

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3686] [Team Posts]
@jekkos I got a gitter message from @odiea saying you needed help with a commit. I have time Tuesday starting at 9am GMT+4. Let me know details and I'll see what I can do.

**Comment by odiea** [comment]
If you don't hear back from him his Blind sql injection fixes (#3284) commit broke lots of tables so just wondering if you have the power to revert that commit

**Comment by objecttothis** [comment]
OK, I'll take a look Tuesday if I don't hear back from him.
